llvm/tools/llvm-dwarfdump/llvm-dwarfdump.cpp

```diff
@@ -295,6 +297,11 @@ static opt<ErrorDetailLevel> ErrorDetails(
            clEnumValN(BothDetailsAndSummary, "full",
                       "Display each error as well as a summary. [default]")),
     cat(DwarfDumpCategory));
+static opt<std::string> JsonSummaryFile(
+    "json-summary-file", init(""),
```

Comment:
My only (minor) concern with this name is that `--statistics` also outputs JSON, but to stdout and that's a bit odd.

Reply:
I'll make the change immediately if you've got a suggestion for a better name.